ee/clickhouse/queries/experiments/funnel_experiment_result.py

```diff
@@ -21,9 +26,7 @@ class Variant:
     failure_count: int
 
 
-CONTROL_VARIANT_KEY = "control"
```

Comment:
Maybe want to move the constant below (`EXPECTED_LOSS_SIGNIFICANCE_LEVEL`) to `__init__.py` too?

Reply:
Was only for funnel_experiments right now (trends have p_value_significance_level, so decided not to move yet.